### ISSUE TITLE
Long executions name pushes the charts

### ISSUE THREAD

**Issue by smunteankestra** [kestra-io/kestra#6226]
### Describe the issue

We might consider adjusting a bit the UI to look more friendly on long executions names

![image](https://github.com/user-attachments/assets/6abdb772-38bc-439b-97d3-fa62a3821cfe)

### Environment

- Kestra Version: develop

**Comment by Ben8t** [comment]
@Nico-Kestra any views on this?

Quick win IMO: change the x-label orientation to the right and align the chart on the left then

**Comment by Nico-Kestra** [comment]
![image](https://github.com/user-attachments/assets/2118f4e2-39c1-4e43-b416-6952d3abe9d0)

If the resolution is smaller, move the second column below the first and stretch the graphs in width.

[FIGMA](https://www.figma.com/design/ew0uXk0NRXJ2NBBJTNe2n1/01_UI?node-id=2411-26518&t=QoYbNXnfe805fzUw-1)

**Comment by MilosPaunovic** [comment]
This would be super difficult to achieve. Maybe to show trimmed down text and tooltips on hover with full label for start?

**Comment by Nico-Kestra** [comment]
The problem is that there could be so many that we’d end up with dots everywhere.

**Comment by Ben8t** [comment]
@MilosPaunovic isn't possible to have one chart per namespace, each one splitted by execution state? The idea of having everything vertical rather than horizontal seems generally a good idea no?

(but if technically too hard, we can look for alternatives)

**Comment by smunteankestra** [comment]
@Ben8t sounds practical in production, yet in automation - we are generate lots of namespaces - idk how it will look

**Comment by Ben8t** [comment]
3 dots issue -> take the last 50 (or less) chars

**Comment by MilosPaunovic** [comment]
So the decision here was, as Ben said, to take last X (let's try with 20) characters from the name of the namespace and show that, with three dots if there is more text, which will show on hover.

**Comment by Ben8t** [comment]
Yes thanks Milos, also please check how that left margin behave in that case (when hovering long text labels)

I'll reevaluate if this chart is really worthwhile too, and not only a "good looking" chart
